scripts/late-for-standup.coffee

```diff
@@ -2,14 +2,21 @@ HubotCron = require 'hubot-cronjob'
 
 random = require '../utils/random.coffee'
 
-CHANCE_TO_BE_LATE = 0.1
+CHANCE_TO_BE_LATE = 0.05
 CHANNEL_ID = 'G0JUTJYCV'
 
+MESSAGES = [
+    '@here Hey team - I'm going to stop at Timmy's next door. Send me your orders.',
```

Comment:
@masneyb there seems to be a syntax error here. Change the outer quotes (') to double quotes (")

Reply:
Corrected. Sorry about that. It was the I'm that caused the issue.